.gitignore

```diff
@@ -1,2 +1,3 @@
+.idea
```

Comment:
Yii Framework is a wonderful thing but not every single aspect of it should be understood as a guide to best practices :P